<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>csfwin</class>
 <widget class="QWidget" name="csfwin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>提取地形点</string>
  </property>
  <widget class="QPushButton" name="para_push">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>布料刚度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>重力参数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="i_rigidness">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="i_timestep">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>迭代次数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="i_iter">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="para_default">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>恢复默认参数</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
